libraries/mongodb.rb

```diff
@@ -22,6 +22,35 @@
 require 'json'
 
 class Chef::ResourceDefinitionList::MongoDB
+  def self.create_replicaset_member(node)
```

Comment:
All of them, because at some point we'd like to be able to support non-chef service discovery. I don't think we need to put a lot of effort into defining the field meanings, just specifying which ones are required, which are optional, and what the defaults are.